README.md

```diff
@@ -37,7 +53,7 @@ GROQ_API_KEY=your-groq-key
 MISTRAL_API_KEY=your-mistral-key
 ```
 
-### 2. Create Workflow File
+### 3. Create Workflow File
 Create `.github/workflows/code-review.yml` with:
 
 ```yaml
```

Comment:
✨ Nice addition of the customize review focus section! This allows users to tailor the review process. 🎯